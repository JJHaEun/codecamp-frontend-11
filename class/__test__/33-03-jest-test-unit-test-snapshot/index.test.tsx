import JestUnitTestPage from "../../pages/section33/33-03-jest-unit-test-snapshot";

import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

it("내가 원하는데로 그려지는지 테스트 하기", () => {
  const result = render(<JestUnitTestPage />); //  가상돔에 그리기
  expect(result.container).toMatchSnapshot(); // 스냅샷이랑 다른게있는지 비교, 없으면 사진을 찍어 만듬.
  // 처음 명령어는 yarn test로 없으면 스냅샷을 찍어둠.. 이후 스냅샷 업데이트 된것으로 찍으려면 yarn test -u
});
