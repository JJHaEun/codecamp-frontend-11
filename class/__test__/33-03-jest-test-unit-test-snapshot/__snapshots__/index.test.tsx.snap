// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`내가 원하는데로 그려지는지 테스트 하기 1`] = `
<div>
  <div>
    철수는 15살 입니다
  </div>
  철수의 취미 입력하기: 
  <input
    type="text"
  />
  <button>
    철수랑 놀러가기
  </button>
</div>
`;
